Network/openvpn.5m.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+# <bitbar.title>MacOS OpenVPN client & DNS Configuration</bitbar.title>
+# <bitbar.version>v0.88008</bitbar.version>
+# <bitbar.author>glowinthedark</bitbar.author>
+# <bitbar.author.github>glowinthedark</bitbar.author.github>
+# <bitbar.desc>OpenVPN GUI & DNS Manager</bitbar.desc>
+# <bitbar.image>https://telegra.ph/file/7e93fd31b281c9cdcab73.png</bitbar.image>
+# <bitbar.dependencies>openvpn</bitbar.dependencies>
+# <bitbar.abouturl>https://github.com/glowinthedark/bitbar-plugins/blob/master/Network/openvpn.5m.sh</bitbar.abouturl>
+
+# *** CONFIGURATION ***
+
+# - INSTALL OPENVPN:
+
+#     brew install openvpn
+# OPTIONAL:
+#     brew install oath-toolkit (for 2FA token generation)
+
+# NOTE: To avoid having to enter the root password allow
+# passwordless sudo for your user on openvpn, killall and pkill:
+
+# 1. Type in terminal: sudo visudo
+# 2. Append the following line at the end of the file:
+
+# replace_with_your_username        ALL = NOPASSWD: /usr/local/sbin/openvpn, /usr/bin/killall, /usr/bin/pkill
+
+# 3. Search and replace all __TODO__ lines with your specific commands
```

Comment:
Would this be a good use case for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html)?

Reply:
@sprak3000: oh! this is something new I was not aware of; in fact, I'm still using the latest tip of the old and discontinued objective-c version; the new go+webview based version uses more CPU on my system, and also with xbar I didn't yet find an equivalent of the previous programmatic refresh with `open -g "bitbar://refreshPlugin?name=openvpn.*?.sh"`; any suggestions on this?

there would probably be quite a few variables to pull into the config dialog; a good challenge for a next iteration :D

Reply:
@glowinthedark 

There is a new `refresh` [parameter](﻿﻿https://github.com/matryer/xbar-plugins/blob/main/CONTRIBUTING.md#parameters). Haven't used it myself or the previous way you mention, so I do not know if they are equivalent.

Thank you for your contribution!